### PR TITLE
Stars was added

### PR DESCRIPTION
**Pull request by RomSmile**
- DEMO LINK: https://RomSmile.github.io/layout_stars/
- TEST REPORT LINK: https://RomSmile.github.io/layout_stars/report/html_report/

### CODE REVIEW

**Comment by RomSmile** [comment]
http://joxi.ru/n2YJJyxi7GXdpA

**Review by MaksymBeliever — APPROVED** [review]
Well done!

**Comment by herkoss** [comment]
Fix please

Снимок экрана 2021-07-08 в 16 11 12

File: src/index.html [hunk 1]
```diff
@@ -8,6 +8,47 @@
     <link rel="stylesheet" href="./style.css">
   </head>
   <body>
-    <h1>Stars</h1>
+    <div class="stars stars--0">
+      <div class="stars_star"></div>
```
Comment (herkoss) [inline src/index.html:12]:
```suggestion
      <div class="stars__star"></div>
```

Comment (herkoss) [reply src/index.html:12]: Here and in a whole file

**Review by herkoss — COMMENTED** [review]
Well done!)

File: src/index.html (hunk 1 above)
Comment (varseniuk) [reply src/index.html:12]: Fix the naming as @herkoss mentioned

**Review by varseniuk — APPROVED** [review]

**Pull request closed without merging**